mailpoet/lib/Automation/Integrations/WooCommerce/Fields/CustomerReviewFieldsFactory.php

```diff
@@ -46,18 +50,27 @@ function (CustomerPayload $payload) {
    * Calculate the customer's review count excluding multiple reviews on the same product.
    * Inspired by AutomateWoo implementation.
    */
-  private function getUniqueProductReviewCount(WC_Customer $customer): int {
+  private function getUniqueProductReviewCount(WC_Customer $customer, int $inTheLastSeconds = null): int {
     $wpdb = $this->wordPress->getWpdb();
+    $inTheLastFilter = isset($inTheLastSeconds) ? "AND comment_date >= DATE_SUB(current_timestamp, INTERVAL %d SECOND)" : '';
```

Comment:
Ahh, thank you! This may explain why I had difficulties testing this manually, while the integration test was always passing. Fixed: https://github.com/mailpoet/mailpoet/pull/5438/commits/3e470fbbe8b0df062303bdc6809085a1cc183abf